src/libraries/System.Security.Cryptography.X509Certificates/tests/CollectionImportTests.cs

```diff
@@ -69,6 +69,7 @@ public static void ImportX509PemFile()
         }
 
         [Fact]
+        [SkipOnPlatform(TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst, "PKCS#7 import is not available of iOS/tvOS/MacCatalyst")]
```

Comment:
A lot of of to on in this file.  They're not really important, since the intention is clear.